graphql/schema/types/external_communications.graphql

```diff
@@ -0,0 +1,85 @@
+input JiraConfigInput {
+  email: String
+  host: String
+  personalAccessToken: String @redactSecrets
+}
+
+type JiraConfig {
+  email: String
+  host: String
+  personalAccessToken: String @requireAdmin
+}
+
+input RuntimeEnvironmentConfigInput {
+    baseurl: String!
+    apiKey: String
+}
+
+type RuntimeEnvironmentConfig {
+    baseurl: String!
+    apiKey: String
+}
+
+input SlackConfigInput {
+  options: SlackOptionsInput
+  token: String! @redactSecrets
+  level: String!
+  name: String!
+}
+
+type SlackConfig {
+  options: SlackOptions
+  token: String @requireAdmin
+  level: String
```

Comment:
i tried to make this an enum but had trouble resolving to `grip` for the priority type